lib/Constants.js

```diff
@@ -1,9 +1,9 @@
 var Constants = {}
-Constants.BILLBOARD_URL = "http://www.billboard.com/charts/hot-100"
-Constants.MAIN_CLASS = "#main div .chart-data .container"
-Constants.SONG_DIV = ".chart-row .chart-row__primary .chart-row__main-display"
-Constants.SONG_NAME = ".chart-row__container .chart-row__title .chart-row__song"
-Constants.SONG_RANK = ".chart-row__rank .chart-row__current-week"
-Constants.SONG_ARTIST =".chart-row__container .chart-row__title .chart-row__artist"
-Constants.SONG_IMAGE = ".chart-row__image"
+Constants.BILLBOARD_URL = "https://www.billboard.com/charts/hot-100"
+Constants.MAIN_CLASS = "#main #charts .charts-container .chart-list.container .chart-list__elements"
+Constants.SONG_DIV = ".chart-list__element .chart-element__wrapper"
+Constants.SONG_NAME = ".chart-element__information .chart-element__information__song"
+Constants.SONG_RANK = ".chart-element__rank .chart-element__rank__number"
+Constants.SONG_ARTIST =".chart-element__information .chart-element__information__artist"
+Constants.SONG_IMAGE = ".chart-element__image"
 module.exports = Constants
```

Comment:
Will verify the ids from the billboard site once and then merge it. Thanks for opening PR, really appreciate the help